### DISCUSSION TITLE
Download and run Setup but I get this error

### DISCUSSION THREAD

**Discussion by ChildeOfShade** [Reloaded-Project/Reloaded-II#512] [General]
I have downloaded the exe and the entire repo but I keep getting this error message. Was there a prerequisite that I'm missing?

![image](https://github.com/user-attachments/assets/8d2c52b6-aec7-46be-a4c4-67aba1fa32f6)

**Comment by Sewer56** [comment]
This is highly unusual, especially since it normally would print an error box with the error details. Not seeing one is unusual.

As a last resort, you can always try manually installing via Release.zip https://github.com/Reloaded-Project/Reloaded-II/releases/download/1.28.5/Release.zip , you may be asked to install a few runtimes, which may be tedious, but it should do the trick.

**Comment by ChildeOfShade** [comment]
Running the EXE in the release seemed to fix it. Not sure what's going on there.

**Comment by ChildeOfShade** [comment]
Downloading the release and running the Reloaded-II.exe file launched the program with no issues. It asked me to download the .NET framework as soon as it launched.
